### PR TITLE
feat: Show members found on backend in start UI

### PR DESCRIPTION
**Pull request by AndyLnd**

### CODE REVIEW

File: src/script/components/userList.ts [hunk 1]
```diff
@@ -175,7 +177,7 @@ ko.components.register('user-list', {
     };
 
     // Filter all list items if a filter is provided
-    this.filteredUserEntities = ko.pureComputed(() => {
+    this.filteredUserEntities = ko.pureComputed(async () => {
```
Comment (ffflorian) [inline src/script/components/userList.ts]: Why is this `async` now?

File: src/script/view_model/list/StartUIViewModel.js [hunk 2]
```diff
@@ -335,25 +336,14 @@ class StartUIViewModel {
     this.services.removeAll();
   }
 
-  _searchPeople(query) {
+  async _searchPeople(query) {
```
Comment (ffflorian) [inline src/script/view_model/list/StartUIViewModel.js:339]: Why is this `async` now?

Comment (AndyLnd) [reply src/script/view_model/list/StartUIViewModel.js:339]: There is an await now in line 373.

Comment (ffflorian) [reply src/script/view_model/list/StartUIViewModel.js:339]: Ah, I see!

**Pull request merged**